### ISSUE TITLE
Codecoverage fails to parse org-noter-core.el

### ISSUE THREAD

**Issue by dmitrym0** [org-noter/org-noter#34]
Undercover fails to parse org-noter-core.el:

Execute `cask exec buttercup -L .` in org-noter directory:

```
UNDERCOVER: Error while loading /Users/dmitrymarkushevich/workspace/org-noter/org-noter-core.el for coverage:
UNDERCOVER: Invalid read syntax: "Expected one of", (symbolp form), (&rest (symbolp form))
UNDERCOVER: The problem may be due to edebug failing to parse the file.
UNDERCOVER: You can try to narrow down the problem using the following steps:
UNDERCOVER: 1. Open "/Users/dmitrymarkushevich/workspace/org-noter/org-noter-core.el" in an Emacs buffer;
UNDERCOVER: 2. Run M-: ‘(require 'edebug)’;
UNDERCOVER: 3. Run M-x ‘edebug-all-defs’;
UNDERCOVER: 4. Run M-x ‘toggle-debug-on-error’.
UNDERCOVER: 5. Run M-x ‘eval-buffer’;
UNDERCOVER: 6. In the *Backtrace* buffer, find a numeric position,
UNDERCOVER:    then M-x ‘goto-char’ to it.
```

Expected: all files should be analyzed
Actual: `org-noter-core.el` fails.

**Comment by dmitrym0** [comment]
I tried to follow the instructions in the error message and I do not get an stack trace.

**Comment by petermao** [comment]
What is `undercover`?  A new testing tool?

**Comment by petermao** [comment]
ah -- PR #33.  Do we hold off merging until this is resolved, or is it easier to deal with if we merge first?

**Comment by dmitrym0** [comment]
That's the one. We should merge, but `undercover` is under-reporting the true coverage as a result of this bug.

**Comment by dmitrym0** [comment]
I looked at this a bit. Turns out that edebug does not know how to deal with macros and needs hints. Here's the [manual link](https://www.gnu.org/software/emacs/manual/html_node/elisp/Instrumenting-Macro-Calls.html) on how to do this. I have used macros so I'm not grokking what the manual is trying to tell me. Have you used macros much, @petermao ?

**Comment by petermao** [comment]
The only macros I've used are the ones in org-noter.  I'll a look at the reference this weekend.

**Comment by dmitrym0** [comment]
Fixed in #79, closing.
